<launch>
  <!-- gogo gazelle wrapper -->
  <node pkg="gogo_gazelle" type="gogo_gazelle" name="gogo_gazelle" output="screen" />
  
  <!-- Run flatten_pointcloud  -->
  <node pkg="floor_octomap" type="flatten_pointcloud_node_0720" name="flatten_pointcloud_node_0720">
    <rosparam command="load" file="$(find floor_octomap)/config/flatten_pointcloud.yaml" />
  </node>
  <!-- footstep planner   -->
  <node pkg="hubo_planner" type="footstep_planning" name="footstep_planning" output="screen">
    <rosparam command="load" file="$(find hubo_planner)/config/footstep.yaml" />
  </node>
  
  <!-- client -->
</launch>
